encrypted-media/clearkey-invalid-license.html

```diff
@@ -0,0 +1,34 @@
+<!doctype html>
+<html>
+<head>
+    <meta charset=utf-8>
+    <meta name="timeout" content="long">
```

Comment:
We don't need this timeout setting. It was originally there in all the previous tests, but should have been removed (very recently).